fastlane/fastlane
# [spaceship] Add support for answering Resolution Center programmatically

### Checklist

### Motivation and Context
Currently it was only possible to `get` information out of the Resolution Center via `app.resolution_center` in `spaceship`. This addition adds the rough capabilities to reply to messages on the resolution center. 

The implementation deliberately lacks the support for uploading and adding files, as this is always quite the hassle. More than happy to review any future pull requests to add this. 

### Description
Short description of the used values:

app_id: Apple ID of the app (e.g. 123456789)
platform: platform of the app (e.g. ios)
thread_id: get the correct thread_id via app.resolution_center
version_id: get the correct version_id via app.resolution_center
version_number: version number
from: your name, can actually be anything, Apple "changes" this automatically
message_body: Your message body. Make sure to escape any special characters and add newlines via \n

### Testing Steps
```
bundle exec irb
require "spaceship"
Spaceship::Tunes.login("test@appleid.com", "password")
app = Spaceship::Tunes::Application.find(123456789)
app.resolution_center // Run this to get information like thread_id, version_id etc.
app.reply_resolution_center(123456789, "ios", "273467226", "832496269", "1.0", "Your Name", "Issues should be fixed now, please review again.")
```
